tests/integration/cairo1/compiler_bindings/starknet_contracts_compiler_test.py

```diff
@@ -6,19 +6,27 @@
 
 def test_contract_to_casm(datadir: Path):
     assert cairo1.compile_starknet_contract_to_casm_from_path(
-        datadir / "basic_starknet_project"
+        input_path=datadir / "basic_starknet_project"
     )
     assert cairo1.compile_starknet_contract_to_casm_from_path(
-        datadir / "test_starknet_project"
+        input_path=datadir / "test_starknet_project"
+    )
+    assert cairo1.compile_starknet_contract_to_casm_from_path(
+        input_path=datadir / "basic_starknet_project",
+        output_path=datadir / "basic_starknet_project.casm",
+    )
+    assert cairo1.compile_starknet_contract_to_casm_from_path(
+        input_path=datadir / "test_starknet_project",
+        output_path=datadir / "test_starknet_project.casm",
```

Comment:
These tests don't really verify if output_path contains anything useful. Is that on purpose?

Reply:
Before recent changes, cairo bindings didn't return anything, when `output_path` was provided. It has changed in recent PRs.
But I have improved these tests in the newest commits